Pod/Classes/URBNAlertView.m

```diff
@@ -111,12 +119,14 @@ - (instancetype)initWithAlertConfig:(URBNAlertConfig *)config alertStyler:(URBNA
         NSDictionary *metrics = @{@"sectionMargin" : self.alertStyler.sectionVerticalMargin,
                                   @"btnH" : self.alertStyler.buttonHeight,
                                   @"lblHMargin" : self.alertStyler.labelHorizontalMargin,
-                                  @"titleVMargin" : titleMargin,
+                                  @"topVMargin" : @([titleMargin doubleValue] + [self.alertStyler.topMargin doubleValue]),
```

Comment:
@bobawhaler what was the logic behind going with `topVMargin` as a combination of `titleMargin` and `topMargin`. How are the two different? What were you not getting out of `titleMargin`. Which is `sectionVerticalMargin`